src/abaqus/XY/XYSession.py

```diff
@@ -9,12 +9,7 @@
 from .XYData import XYData
 from .XYSessionBase import XYSessionBase
 from ..PathAndProbe.Path import Path
-
-
-# prevent circular imports
-class Odb:
-    pass
-
+from ..Odb.Odb import Odb
```

Comment:
I think this New Odb object was made because of the circular imports, have you checked it?

Reply:
No, I really haven't check it properly, sorry. This correction was made to make the class with same namespace, since the typechekcer was complaining about that with warnings.

Reply:
I haven't found a proper way to solve this problem, so I leave it this way, although it is not pretty.